staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -125,6 +125,13 @@ func createHandler(r rest.NamedCreater, scope RequestScope, admit admission.Inte
 			}
 		}
 
+		laa := NewLastAppliedAccessor(workflowID, *gvk, obj)
+
+		if err := laa.SaveNew(body, obj); err != nil {
```

Comment:
If I'm reading this correctly, this is assuming the submitted object is json or yaml, and errors if it isn't. That won't work with objects submitted in protobuf format (which is what controllers and kubelets do by default)

 It is also adding a new decoding path to the apiserver, meaning we're not using the existing content-type handling, and we're double decoding.

Reply:
Would it make more sense to try to capture the pre-conversions/defaulting object decoded from the response?

Reply:
Yes, the failing test due to this already suggested using already present decoding.
In Slack @apelisse just said that we may not want to store the defaulted object so what would be your opinion on that?